modules/loader-utils/src/lib/binary-utils/memory-copy-utils.js

```diff
@@ -1,16 +1,9 @@
+import assert from '../env-utils/assert';
 import {sliceArrayBuffer} from './array-buffer-utils';
 
-export function padTo4Bytes(byteLength) {
-  return (byteLength + 3) & ~3;
-}
-
-export function padToNBytes(byteLength, padding = 4) {
-  if (byteLength < 0) {
-    throw new Error(`Incorrect 'byteLength' value: ${byteLength}`);
-  }
-  if (padding <= 0) {
-    throw new Error(`Incorrect 'padding' value: ${padding}`);
-  }
+export function padToNBytes(byteLength, padding) {
+  assert(byteLength >= 0, `Incorrect 'byteLength' value: ${byteLength}`);
```

Comment:
We intentionally don't include error strings in the asserts as this bloats the user bundle size but only benefits developers.
```suggestion
  assert(byteLength >= 0,); // `Incorrect 'byteLength' value: ${byteLength}`
```
The developer can still break on exceptions and read the comment here.